conf.yaml

```diff
@@ -388,8 +388,8 @@ contents:
               -
                 title:      PHP API
                 prefix:     php-api
-                current:    6.5.x
-                branches:   [ master, 6.5.x, 5.0, 2.0, 1.0, 0.4 ]
+                current:    6.0
+                branches:   [ master, 6.0, 5.0, 2.0, 1.0, 0.4 ]
```

Comment:
If you still want to build docs for 6.5.x, you'll need to include it in the `branches` value.

```suggestion
                branches:   [ master, 6.5.x, 6.0, 5.0, 2.0, 1.0, 0.4 ]
```

After merging, the dropdown should look like this:
<img width="214" alt="Screen Shot 2019-04-24 at 8 41 09 AM" src="https://user-images.githubusercontent.com/40268737/56660112-c5a5f680-666c-11e9-8741-da75d7af45d2.png">

Reply:
@jrodewig I need to have 6.0 instead of 6.5.x because this is the old behavior and I don't want to change the expectation. I did a 6.5.x branch because of some API changes but I figured out how to manage it in a consistent way without BC breaks. That means I'll remove the 6.5.x branch when this PR will be merged with 6.0 branch.

Reply:
Ah. I understand. Please disregard my suggestion then.